spec/requests/api/v1/events/nearest_spec.rb

```diff
@@ -0,0 +1,64 @@
+describe 'Events Requests' do
+  describe 'GET /v1/events/nearests?lat=&lon=&radius=' do
+    it 'returns events closest to lat and lon' do
+      near_event = create(:event,
+                          lat: 37.760322,
+                          lon: -122.429667)
+      farther_event = create(:event,
+                             lat: 37.760321,
+                             lon: -122.429667)
+      create(:event, lat: 37.687737, lon: -122.470608)
```

Comment:
Also consider having factory aliases for [:far, :near, :farthset] instead of having to inline create the values.

Well, even :boston, :la, :chicago as aliases could be helpful.

Then you can quickly see in tests what really should be far and some sense of distance.

Could come in handy when making most tests.